docs/index.rst

```diff
@@ -28,6 +26,30 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available Solidity Integrations
+-------------
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+	WEB compiler for solidity language which could also simulate function calls.	
+
+* `Ethereum Studio <https://live.ether.camp/>`_
+	A fully-functional well-designed WEB IDE for Ethereum Solidity developers.
+
+* `Visual Studio Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
+	Consensys integration of Solidity into Microsoft Visual Studio 2015
+
+* `Mix IDE <https://github.com/ethereum/mix/>`_
+	A developer's IDE for designing, debugging and testing solidity smart contracts.
+
+* `Package for SublimeText — Solidity language syntax <https://packagecontrol.io/packages/Ethereum/>`_
+	Solidity syntax highlight for SublimeText editor.
+
+* `Atom Solidity package <https://github.com/gmtcreators/atom-solidity/>`_
+	Analog of solidity browser-based compiler for atom editor.
+
+* `Visual Studio Code extension <http://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
+	Solidity syntax highlight and debugging for Visual Studio Code.
```

Comment:
`Solidity plugin for Microsoft Visual Studio Code that includes syntax hilighting and the Solidity compiler.`